.jenkins/pipelines/standalone/azure-sdk-tests.Jenkinsfile

```diff
@@ -99,8 +102,49 @@ pipeline {
                    """
             }
         }
+        stage("Install package") {
+            when {
+                expression { params.PACKAGE_NAME != "" }
+            }
+            steps {
+
+                azureDownload(
+                    downloadType: 'container',
+                    containerName: 'mystikosreleases',
+                    includeFilesPattern: "${PACKAGE_INSTALL}",
```

Comment:
If `PACKAGE_NAME` isn't defined (which is a condition of running this stage) then `PACKAGE_INSTALL` would produce an invalid name? E.g. `Ubuntu-2004_.deb`

Reply:
I moved the package install step into a stage requiring `PACKAGE_NAME` to be a non-empty string